transforms/universal/ededup/python/README.md

```diff
@@ -0,0 +1,33 @@
+# Ededup Python Transform 
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+This is a python version of ededup
+
+## Configuration and command line Options
+
+The set of dictionary keys holding [EdedupPythonTransform](src/ededup_transform_python.py) 
+configuration for values are as follows:
+
+* _ededup_doc_column_ - specifies the name of the column containing a document
+the `transform()` method of `EdedupPythonTransform`. 
```

Comment:
seems like something got dropped here

Reply:
fixed

Reply:
Still unreadable.  Suggest
`
specifies the name of the column containing a document that
the `transform()` method of `EdedupPythonTransform` examines for duplicates 
among other documents. `